dvc/commands/experiments/push.py

```diff
@@ -18,34 +19,60 @@ def raise_error_if_all_disabled(self):
                 "`--rev` or `--all-commits` flag."
             )
 
-    def run(self):
+    @staticmethod
+    def log_result(result: Dict[str, Any], remote: str):
         from dvc.utils import humanize
 
-        self.raise_error_if_all_disabled()
+        def join_exps(exps):
+            return humanize.join([f"[bold]{e}[/]" for e in exps])
 
-        result = self.repo.experiments.push(
-            self.args.git_remote,
-            self.args.experiment,
-            all_commits=self.args.all_commits,
-            rev=self.args.rev,
-            num=self.args.num,
-            force=self.args.force,
-            push_cache=self.args.push_cache,
-            dvc_remote=self.args.dvc_remote,
-            jobs=self.args.jobs,
-            run_cache=self.args.run_cache,
-        )
-        if pushed_exps := result.get("pushed"):
-            ui.write(
-                f"Pushed experiment {humanize.join(map(repr, pushed_exps))} "
-                f"to Git remote {self.args.git_remote!r}."
+        if diverged_exps := result.get("diverged"):
+            exps = join_exps(diverged_exps)
+            ui.warn(
+                f"Local experiment {exps} has diverged "
+                "from remote experiment with the same name. "
+                "To override the remote experiment re-run with '--force'."
             )
-        else:
+        if uptodate_exps := result.get("up_to_date"):
+            verb = "are" if len(uptodate_exps) > 1 else "is"
+            exps = join_exps(uptodate_exps)
+            ui.write(f"Experiment {exps} {verb} up to date.", styled=True)
```

Comment:
Sometimes the first letter gets pushed to the end of the line, investigating why it's happening, something to do with Tqdm. :(

Reply:
Haven't noticed it yet FWIW. 

Can we make the message `Experiment {exps} {verb} up to date on Git remote {remote}`? It would make clear we are talking about Git being up to date rather than the DVC cache, and it would be more similar to the message below for pushed experiments.